src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/core/stats/eval/impl/vals/Value.java

```diff
@@ -0,0 +1,21 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.core.stats.eval.impl.vals;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.core.stats.eval.Statistics;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.core.stats.format.Formatter;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.core.stats.measurements.MeasurementSet;
+
+public class Value {
```

Comment:
This class doesnt seem to have anything substantial. Do we really need it?

Reply:
There are certain statistics that use the value type. They only contain the numeric values